helm/values.yaml

```diff
@@ -59,27 +60,27 @@ secret:
 #   # See https://helm.fiftyone.ai/docs/expose-teams-api.html for more information.
 #   dnsName: your-api.hostname.here
 #   env:
-#     Set FIFTYONE_PLUGINS_DIR if you are enabling plugins in a dedicated
+#     # Set FIFTYONE_PLUGINS_DIR if you are enabling plugins in a dedicated
 #       `teams-plugins` deployment
-#     See https://github.com/voxel51/fiftyone-teams-app-deploy/tree/main/helm/fiftyone-teams-app#fiftyone-teams-plugins
+#     # See https://github.com/voxel51/fiftyone-teams-app-deploy/tree/main/helm/fiftyone-teams-app#fiftyone-teams-plugins
 #     FIFTYONE_PLUGINS_DIR: /opt/plugins
-#     Set FIFTYONE_TEAMS_VERSION_OVERRIDE to override the `Install FiftyOne`
-#       bash command in the `Settings > Install FiftyOne` modal
+#     # Set FIFTYONE_TEAMS_VERSION_OVERRIDE to override the `Install FiftyOne`
+#     # bash command in the `Settings > Install FiftyOne` modal
```

Comment:
These allow for a user to uncomment the block and still have the comments for the non-config values.